backend/novion.py

```
@@ -121,20 +296,148 @@ def researcher_node(state: State) -> Command[Literal["supervisor"]]:
+    Args:
+        enabled: If True, MCP tools will be enabled. If False, they will be disabled.
+        
+    Returns:
+        A message indicating the current state of MCP integration.
+    """
+    global pharamcist_agent, researcher_agent, medical_analyst_agent
+    global pharamcist_base_agent, researcher_base_agent, medical_analyst_base_agent
+    
+    if enabled:
+        # Re-enable MCP for all agents
+        pharmacist_agent = enhance_agent_with_mcp(
```

Ruff (0.8.2)

Local variable `pharmacist_agent` is assigned to but never used

Remove assignment to unused variable `pharmacist_agent`

(F841)